kuma/wiki/templates/wiki/document.html

```diff
@@ -416,7 +416,10 @@ <h2 class="offscreen">{{ _('Document Tags and Contributors') }}</h2>
       {# This can be removed once the social network button multivariate test is completed. #}
       mdn.optimizely.push(['activate', 2796520257]);
   </script>
-  {% if waffle.flag('share_links') or waffle.flag('helpfulness') %}
-    {{ js('document') }}
+  {% if waffle.flag('share_links') %}
+    {{ js('social') }}
+  {% endif %}
```

Comment:
Both/either/neither are all valid. That's why the original code didn't work -- it coupled them.